db/models/articles.models.js

```diff
@@ -14,6 +15,22 @@ function fetchArticleById(article_id){
         }
           return article
         })
-}
 
-module.exports = { fetchArticleById }
\ No newline at end of file
+}
+    exports.fetchAllArticles = ()=>{
+        const queryStatment = `
+        SELECT articles.*, COUNT(comments.article_id) FROM articles
+        LEFT JOIN comments ON articles.article_id = comments.article_id
+        GROUP BY (articles.article_id)
+        ORDER BY article_id DESC;`
+        return db
```

Comment:
You will need to change the query a bit to ensure the body isn't part of the object